Dashboard/app/js/lib/views/surveys/survey-group-views.js

```diff
@@ -82,7 +82,7 @@ FLOW.Project = FLOW.View.extend({
 
   isPublished: function() {
     var form = FLOW.selectedControl.get('selectedSurvey');
-    return form && form.get('status') === 'PUBLISHED'
+    return form.get('status') === 'PUBLISHED'
   }.property('FLOW.selectedControl.selectedSurvey.status'),
```

Comment:
Discussed on Skype:

> The rollback was fixing the display of a blank screen... but the root cause of that is some convoluted system of state management with lots of global variables that at some point go out of sync and there is no survey data

> addressing the symptom by checking for a null form solves the appearance.. i.e. no blank screen but later on results in issues like options missing from questions or question types incorrectly set etc... and I felt that is way more dangerous because then a user may save the wrong settings on a question and we are in trouble

> therefore rollback... let the blank page issue persist that forces user to reload the page and have correct state... and then we can sort out the proper state management issues later